hq/markdown/index.md

```diff
@@ -0,0 +1,3 @@
+# List of help pages
+
+   1. How to implement [Snyk](snyk) Vulnerability Analysis
```

Comment:
Am I missing something here, or should `(snyk)` be a url?

Reply:
It is a _relative_ url.